server/protocols/jmap-draft/src/main/java/org/apache/james/jmap/draft/methods/GetMailboxesMethod.java

```diff
@@ -136,6 +140,7 @@ private GetMailboxesResponse getMailboxesResponse(GetMailboxesRequest mailboxesR
 
     private Stream<Mailbox> retrieveSpecificMailboxes(MailboxSession mailboxSession, ImmutableList<MailboxId> mailboxIds) throws MailboxException {
         List<MessageManager> mailboxes = mailboxManager.getMailboxes(mailboxIds, mailboxSession);
+        Map<MailboxId, MailboxCounters> mailboxCounters = retrieveMailboxCounters(mailboxSession, mailboxes.stream().map(MessageManager::getId).collect(Guavate.toImmutableList()));
```

Comment:
Can't we just use `mailboxIds` instead of 
`mailboxes.stream().map(MessageManager::getId).collect(Guavate.toImmutableList())` ?